python/Ganga/__init__.py

```diff
@@ -309,7 +309,7 @@ def getLCGRootPath():
                  filter=Ganga.Utility.Config.expandvars)
 
 lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
+    'GLITE_ENABLE', False, 'Enables/disables the support of the GLITE middleware')
```

Comment:
This is quite a significant change that will probably annoy a lot of people. Can't it be put in a the appropriate .ini file instead?